doc/src/index.md

```diff
@@ -1,3 +1,36 @@
+```@raw html
+---
+layout: home
+
+hero:
+  text: "The Julia Programming Language"
+  image:
+    light:
+        src: /logo.svg
+        alt: julia
+    dark:
+        src: /logo-dark.svg
+        alt: julia
+  actions:
+    - theme: brand
+      text: Getting Started
+      link: /manual/getting-started
+    - theme: alt
+      text: View on Github
+      link: https://github.com/JuliaLang/julia
+    - theme: brand
+      text: Sponsor
+      link: https://github.com/sponsors/julialang
+---
+```
```

Comment:
This is.. wrong? It's not HTML. I guess we could have an `@raw vitepress` maybe. But having to hack this information in like this seems wrong regardless. Seems like it should maybe be an at-meta block? That would require adding some interfaces to Documenter I think.

Reply:
I think this is actually YAML, but a general interface for arbitrary front matter would be pretty great!

Reply:
perhaps 
```
Frontmatter = """
my raw frontmatter here...
"""
```
might work?